backend/routes/admin.py

```
@@ -886,3 +902,843 @@ async def get_all_users(session: Dict[str, Any] = Depends(require_admin_role)) -
+) -> Dict[str, Any]:
+    """Update follow-up question settings."""
+    try:
+        # Validate and create settings object
+        settings = FollowUpSettings.from_dict(settings_data)
+
+        # Use settings manager to store settings
+        settings_mgr = get_settings_manager()
+        success = settings_mgr.set_followup_settings(settings, session["user_id"])
+
+        if not success:
+            raise HTTPException(status_code=500, detail="Failed to update settings")
```

Ruff (0.12.2)

Abstract `raise` to an inner function

(TRY301)